website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md

```diff
@@ -208,6 +208,67 @@ the integration between Okta and dbt Cloud.
 3. After creating the Okta application, follow the instructions in the [dbt Cloud Setup](#dbt-cloud-setup)
 section to complete the integration.
 
+## Google integration
+
+_Use this section if you are configuring Google as your identity provider_.
+
+### Configure the Google application
+
+Sign into your **Google Admin Console** via an account with super administrator privileges.
+
+* From the Admin console Home page, go to **Apps** > **Web and mobile apps**.
+* Click **Add** > **Add custom SAML app**.
+
+Click **Next** to continue.
+
+On the **App Details** page:
+
+* Name the custom app
+* Upload an app logo (optional)
+* Click **Continue**.
+
+### Configure SAML Settings
+
+On the **Google Identity Provider details** page:
+
+* Download the **IDP metadata**.
+* Copy the **SSO URL** and **Entity ID** and download the **Certificate** (or **SHA-256 fingerprint**, if needed).
+
+On the **Service Provider Details** window, enter the following values:
+
+* **ACS URL**: `https://cloud.getdbt.com/complete/saml`
+* **Audience URI (SP Entity ID)**: `https://cloud.getdbt.com/`
+* **Start URL**: (if needed)
+
+The default **Name ID** is the primary email. Multi-value input is not supported.
+
+Use the **Attribute mapping** page to
+map your organization's Google Directory Attributes to the format that
+dbt Cloud expects; click **Add another mapping** to map additional attributes.
+
+Expected **Attributes**:
+
+| Name           | Name format | Value                | Description                |
+| -------------- | ----------- | -------------------- | -------------------------- |
+| `First name`   | Unspecified | `first_name`         | _The user's first name_    |
```

Comment:
We should get rid of these italics because they're not necessary.
```suggestion
| `First name`   | Unspecified | `first_name`         | The user's first name.  |
```